src/services/carrier_type_service.js

```diff
@@ -12,8 +12,10 @@ export default (easypostClient) =>
      * @returns {Array|Promise<never>}
      */
     static async all(query = {}) {
+      const url = `${this._url}`;
```

Comment:
All occurances of ``${this._url}`` should just be `this._url` since there are no other strings as a part of it. Then, if we did that, not really a reason to assign it to a url variable but instead could just pass it in to the HTTP call directly

Reply:
I agree with the first part about string interpolation. The second part I'll counter with readability and consistency.